opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -1844,7 +1866,7 @@ void FieldProps::init_porv(Fieldprops::FieldData<double>& porv)
     }
 
     for (const auto& mregp: this->multregp) {
-        const auto& index_list = this->region_index(mregp.region_name, mregp.region_value);
+        const auto& index_list = this->region_index(mregp.region_name, mregp.region_value).first;
```

Comment:
I don't recall the exact details here, but this looks a little hazardous to me.  I don't think we can depend on lifetime extension for parts of an object.  In this case I'd probably just write
```C++
const auto index_list = this->region_index(...).first;
```
(i.e., make `index_list` be an object rather than a reference) instead.